Automattic/woocommerce-payments
# Add helper to get request button handler class instance.

Fixes #2973 (short-term patch)

#### Changes proposed in this Pull Request

Add a class method to return the `WC_Payments_Payment_Request_Button_Handler` instance.

In the very short term, this would allow me to remove the payment request buttons on a product that uses Name Your Price since request buttons are not yet capable of getting the dynamically entered price.

#### Testing instructions

This snippet should remove the payment request buttons on simple products. 🤞 
```
/**
 * Remove Stripe's payment request buttons
 */
function kia_remove_request_buttons() {
    
    global $product;

    if ( is_callable( array( 'WC_Payments', 'get_payment_request_button_handler' ) ) && $product instanceof WC_Product && $product->is_type( 'simple' ) ) {
        remove_action( 'woocommerce_after_add_to_cart_quantity', [ WC_Payments::get_payment_request_button_handler(), 'display_payment_request_button_html' ], 1 );
        remove_action( 'woocommerce_after_add_to_cart_quantity', [ WC_Payments::get_payment_request_button_handler(), 'display_payment_request_button_separator_html' ], 2 );
    }

}
add_action( 'woocommerce_after_add_to_cart_quantity', 'kia_remove_request_buttons', -1 );
```
